tron/kubernetes.py

```diff
@@ -337,11 +337,12 @@ def handle_next_event(self, _=None) -> None:
         * control: events regarding how the task_proc plugin is running - handled directly
         * task: events regarding how the actual tasks/Pods we're running our doing - forwarded to KubernetesTask
         """
-        if self.deferred and not self.deferred.called:
+        if self.deferred is not None and not self.deferred.called:
             log.warning("Already have handlers waiting for next event in queue, not adding more")
             return
 
         self.deferred = self.queue.get()
+        assert self.deferred is not None
```

Comment:
i'm 99.99% sure this assert will never trigger - but just to be safe:
```suggestion
        if self.deferred is None:
            log.warning("Unable to get a handler for next event in queue - this should never happen!")
            return
```
should make mypy happy as well as ensure that we don't crash the whole process should the impossible be possible :)

Reply:
yup that is a much more elegant solution 😅